### ISSUE TITLE
doubts regarding end Result

### ISSUE THREAD

**Issue by Lazeeez** [HousewareHQ/data-engineering-octernship#5]
Hey guys, I had a question regarding end result of this assignment.

Are we supposed to just code the whole assignment in python and create a API endpoint (I'm using Flask) locally or do we have to host it on a server and then submit the link?

@shubh24 @ashwinexe

**Comment by Lazeeez** [comment]
Currently, I wrote the python script to load the data from .csv to .db and then perform the operations on that .db file and create another .csv file (different files for different criteria) and save the result of those criteria in those files.

The script takes input from a .json file in the same directory and I also made the script very generalized so that it can take various different operations and not just the given criteria.

**Comment by gaurav-2-0-0-2** [comment]
I have a question did you run the converted query on your database ?

**Comment by Lazeeez** [comment]
@gaurav-2-0-0-2 yes, it prints out the result and also stores it into an output file provided in the .json

**Comment by gaurav-2-0-0-2** [comment]
where are you printing those results in the console or at the page itself.

**Comment by Lazeeez** [comment]
@gaurav-2-0-0-2 it basically returns the result in the message, the return statement of the API

**Comment by gaurav-2-0-0-2** [comment]
Oh, I'm actually using node.js express for writing the API and posting the request (the JSON data) via postman.

**Comment by gaurav-2-0-0-2** [comment]
Are you using any tool like knex.js to convert JSON data into query or just the python code.

**Comment by Lazeeez** [comment]
core python and some other libraries

**Comment by Lazeeez** [comment]
> Oh, I'm actually using node.js express for writing the API and posting the request (the JSON data) via postman.

did you host it somewhere orr are you running it locally?

**Comment by gaurav-2-0-0-2** [comment]
> did you host it somewhere orr are you running it locally?
I'm actually running it locally. I think they should also provide the info about it. I'm confused too.
